meta-mel-support/recipes-bsp/96boards-tools/96boards-tools_0.7.bb

```diff
@@ -26,7 +27,18 @@ do_install () {
 
     install -d ${D}${sbindir}
     install -m 0755 ${S}/resize-helper ${D}${sbindir}
+
+	install -d ${D}${sysconfdir}/init.d
+	install -m 0755 ${WORKDIR}/resize-helper.sh ${D}${sysconfdir}/init.d/resize-helper.sh
+
+	sed -i -e "s:udevadm settle:$bindir\/udevadm settle:" ${D}${sysconfdir}/init.d/resize-helper.sh
+	sed -i -e "0,/resize-helper/ s:resize-helper:$sbindir\/resize-helper:" ${D}${sysconfdir}/init.d/resize-helper.sh
+	sed -i -e "s:rc3:$sysconfdir\/rc3:" ${D}${sysconfdir}/init.d/resize-helper.sh
+	sed -i -e "s:rc5:$sysconfdir\/rc5:" ${D}${sysconfdir}/init.d/resize-helper.sh
```

Comment:
@kergoth 
updated $sysconfdir with ${sysconfdir} & used the suggested placeholder, thanks